Makefile

```
@@ -53,14 +53,16 @@ PB2PYS  = $(PROTOS:$(PROTO_ROOT)/%.proto=$(PB2DIR_ROOT)/$(SHADOW_ROOT)/%_pb2.py)
 PB2PY_VALIDATE = $(PB2DIR_ROOT)/validate/validate_pb2.py
 PB2PY_GOOGLEAPIS = $(PB2DIR_ROOT)/googleapis/googleapis/google/api/annotations_pb2.py
 PB2PY_GOOGLERPCS = $(PB2DIR_ROOT)/googleapis/googleapis/google/rpc/status_pb2.py
+PB2PY_VTPROT = $(PB2DIR_ROOT)/vtproto/ext_pb2.py
```

I think this file is not proto so how about naming like below?
```suggestion
PB2PY_VTEXTPY = $(PB2DIR_ROOT)/vtproto/ext_pb2.py
```

---

Thank you for your comment. It looks good, so I will fix 🙏